src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,32 +2,35 @@
 
 public class DefaultCountingOutRhymer {
 
-    private int[] numbers = new int[12];
+    public static final int INT = -1;
+    public static final int INT1 = 11;
+    public static final int INT2 = 12;
+    private final int[] numbers = new int[INT2];
 
-    public int total = -1;
+    public int total = INT;
 
     public void countIn(int in) {
         if (!isFull())
             numbers[++total] = in;
     }
 
     public boolean callCheck() {
-        return total == -1;
+        return total == INT;
     }
 
     public boolean isFull() {
-        return total == 11;
+        return total == INT1;
     }
 
     protected int peekaboo() {
         if (callCheck())
-            return -1;
+            return INT;
         return numbers[total];
     }
 
     public int countOut() {
         if (callCheck())
-            return -1;
+            return INT;
```

Comment:
I really don't like the fact that we use this static variable in the return value, but I understand that you wanted to reduce the amount of code